### PR TITLE
PARQUET-540: Fix Cascading 3 build thrift and SLF4J.

### PR DESCRIPTION
**Pull request by rdblue**
This fixes:
* parquet-cascading3 should have a libthrift dependency that uses thrift.version
* parquet-cascading3 should have the standard SLF4J dependencies
* Twitter's maven repo is no longer necessary because Parquet uses the Apache maven-thrift-plugin

### CODE REVIEW

**Comment by rdblue** [comment]
@danielcweeks, can you take a look at this? Thanks!

**Comment by rdblue** [comment]
@julienledem, can you look at this?

Cascading 3 broke the build for thrift 9. This fixes the problem. Thanks!

File: parquet-cascading/pom.xml [hunk 1]
```diff
@@ -157,7 +157,7 @@
       <plugin>
         <groupId>org.apache.thrift.tools</groupId>
         <artifactId>maven-thrift-plugin</artifactId>
-        <version>0.1.10</version>
+        <version>0.1.11</version>
```
Comment (piyushnarang) [inline parquet-cascading/pom.xml]: pull this version constant out to the root pom file?

**Comment by piyushnarang** [comment]
minor comment, apart from that lgtm

**Comment by rdblue** [comment]
@julienledem, since you're reviewing today, could you take a quick look at this one? It's a big annoyance. :)

**Comment by danielcweeks** [comment]
+1

**Pull request closed without merging**